modules/network-observability-viewing-network-events.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-viewing-network-events_{context}"]
+= Viewing network events
+:FeatureName: Viewing `NetworkEvents`
+include::snippets/technology-preview.adoc[]
+
+You can view network events, such as network flows that are dropped or allowed by a network policy, by editing the `FlowCollector` to the specifications in the following YAML example.
+
+.Prerequisites
+* Must have `OVNObservability` enabled by enabling the `TechPreviewNoUpgrade` feature set in the `FeatureGate` CR named `cluster`. For more information, see "Enabling feature sets using the CLI" and "Checking OVN-Kubernetes network traffic with OVS sampling using the CLI" in the "Additional resources" of this section.
+* Must have a network policy enabled to see network events for network policy, as shown in the Verification. 
```

Comment:
@memodi @msherif1234 does it help to add to the prerequisites that they must have a network policy enabled to be able to see network policy events?

Reply:
I removed the prerequisite add after discussion in SCRUM and instead adjusted the example output in the verification to be more inclusive and flexible for different output scenarios.

Reply:
> I know we were discussing whether or not this should be a prerequisite. Maybe it should, since it is in the OVN-K docs?

Ok, let's follow them then. My rationale was that you can turn on the feature even without having any policy defined - and the network events _still_ tell you meaningful information by their absence - that there is **no** policy installed. But let's be consistent with ovnk doc ;-)